AEPServices/Tests/services/DiskCacheServiceTests.swift

```diff
@@ -184,3 +215,16 @@ class DiskCacheServiceTests: XCTestCase {
         XCTAssertNil(diskCache.get(cacheName: CACHE_NAME, key: ENTRY_KEY))
     }
 }
+
+private extension Date {
+    func equal(other: Date?) -> Bool {
+        guard let other = other else {
+            return false;
+        }
+
+        // As date stores the timestamp in milliseconds, compare double value with accuracy greater than milliseconds.
+        let accuracy = 0.00001;
```

Comment:
wouldn't this just be 0.001?  my two cents is we could make it even a little more forgiving than this so we don't have random failures with this equality check